data/attributes.yml

```diff
@@ -32352,6 +32352,27 @@ base_attributes:
   - broadhead_design__fixed_blade
   - broadhead_design__mechanical
   - broadhead_design__other
+- id: 4678
+  name: Scooter brake type
+  description: Specifies the mechanism and design of scooter brakes, such as disc or drum
+  friendly_id: scooter_brake_type
+  handle: scooter-brake-type
+  values:
+  - scooter_brake_type__combination_brakes
+  - scooter_brake_type__disc_brakes
+  - scooter_brake_type__drum_brakes
+  - scooter_brake_type__foot_brakes
+  - scooter_brake_type__regenerative_brakes
+  - scooter_brake_type__other
```

Comment:
Ideally, we should remove the word "brakes" from these values to keep things consistent (see `Bicycle brake type` as an example).